core/src/main/res/values/arrays.xml

```diff
@@ -1,6 +1,12 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
 
+    <string-array name="spnAutoDeleteItems">
+        <item>Global</item>
+        <item>Always</item>
+        <item>Never</item>
```

Comment:
Sure, I'll do that.
Out of curiosity, was it possible to simply build a tree of localized arrays.xml instead?
I guess since 90% of array.xml doesn't need to be localized, it was chosen to put all the localization in one file?
https://developer.sinnerschrader-mobile.com/moving-localization-from-arrays-xml-to-strings-xml/463/